cli/endpoints/online/managed/vnet/onlinescoring/score.py

```diff
@@ -0,0 +1,35 @@
+import os
+import logging
+import json
+import numpy
+import joblib
+
+
+def init():
+    """
+    This function is called when the container is initialized/started, typically after create/update of the deployment.
+    You can write the logic here to perform init operations like caching the model in memory
+    """
+    global model
+    # AZUREML_MODEL_DIR is an environment variable created during deployment.
+    # It is the path to the model folder (./azureml-models/$MODEL_NAME/$VERSION)
+    model_path = os.path.join(
+        os.getenv("AZUREML_MODEL_DIR"), "sklearn_regression_model.pkl"
+    )
+    # deserialize the model file back into a sklearn model
+    model = joblib.load(model_path)
+    logging.info("Init complete")
+
+
+def run(raw_data):
+    """
+    This function is called for every invocation of the endpoint to perform the actual scoring/prediction.
+    In the example we extract the data from the json input and call the scikit-learn model's predict()
+    method and return the result back
+    """
+    logging.info("Request received")
+    data = json.loads(raw_data)["data"]
+    data = numpy.array(data)
+    result = model.predict(data)
+    logging.info("Request processed")
```

Comment:
minor: Do you want to log the request contents as well?

Reply:
Like discussed, it is fine without that.